specs/darwin/power_sensors.table

```diff
@@ -1,7 +1,7 @@
 table_name("power_sensors")
 description("Machine power (currents, voltages, wattages, etc) sensors.")
 schema([
-    Column("key", TEXT, "The SMC key on macOS", index=True),
+    Column("key", TEXT, "The SMC key on macOS", index=True, optimized=True),
```

Comment:
`genPowerSensors` is so nested it's hard to traverse all the bits, but I _think_ it's optimized.